docs/errorhandling.rst

```diff
@@ -28,6 +28,59 @@ exception to the :attr:`~flask.Flask.logger`.
 But there is more you can do, and we will cover some better setups to deal
 with errors.
 
+.. _error-handlers:
+
+Error handlers
+--------------
+
+.. versionadded:: 1.0
+
+Some errors are unavoidable because not only you make them.  Users might
```

Comment:
Right, but I find it unnecessary to introduce the reader to the idea that apps may throw errors *twice*. I find it barely necessary the first time, and regardless whether it's a user or server error, you probably want to register errorhandler for either, so there's no real distinction regarding usecases.